pyfolio/plotting.py

```diff
@@ -94,10 +96,10 @@ def context(context='notebook', font_scale=1.5, rc=None):
                                 rc=rc)
 
 
-def plot_rolling_risk_factors(
+def plot_rolling_fama_french(
         returns,
         risk_factors=None,
-        rolling_beta_window=63 * 2,
+        rolling_window=BDAYS_PER_MONTH * 6,
```

Comment:
Yeah, I changed it to be 6 months throughout as I'm worried about stability of the estimates. 126 days should be pretty robust while 63 is certainly not enough to get reliable regression estimates. @justinlent what do you think?